admin/src/views/RecipientsView.tsx

```diff
@@ -0,0 +1,43 @@
+import { EntityCollectionView, useAuthController } from '@camberi/firecms';
+import { EntityCollection } from '@camberi/firecms/dist/types';
+import { ToggleButton, ToggleButtonGroup } from '@mui/lab';
+import { Box } from '@mui/material';
+import { RECIPIENT_FIRESTORE_PATH } from '@socialincome/shared/src/types';
+import { useState } from 'react';
+import { buildRecipientsCollection, buildRecipientsRecentPaymentsCollection } from '../collections';
+
+type RecipientViewOptions = 'all' | 'recentPayments';
+
+export function RecipientsView() {
+	const authController = useAuthController();
+	const isGlobalAdmin = !!authController.extra?.isGlobalAdmin;
```

Comment:
what is the `!!` doing?

Reply:
It's simply a double negation. Makes sure that the value is actually a boolean and not `undefined` if no value is passed.